core-common/src/main/java/org/zalando/nakadi/annotations/validation/LabelKey.java

```diff
@@ -0,0 +1,32 @@
+package org.zalando.nakadi.annotations.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({
```

Comment:
Is there difference from `AnnotationKey` validatior?

Reply:
No